stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerView.kt

```diff
@@ -138,6 +172,53 @@ public class MessageComposerView : ConstraintLayout {
         (binding.trailingContent.children.first() as? MessageComposerChild)?.renderState(state)
         (binding.centerContent.children.first() as? MessageComposerChild)?.renderState(state)
         (binding.leadingContent.children.first() as? MessageComposerChild)?.renderState(state)
+
+        if (state.mentionSuggestions.isNotEmpty()) {
+            renderMentionSuggestions(state)
+        } else {
+            suggestionsPopup?.dismiss()
+        }
+    }
+
+    /**
+     * Displays list of mention suggestions, or updates it according to the [MessageInputState.mentionSuggestions] list.
+     *
+     * @param state Current instance of [MessageInputState].
+     */
+    private fun renderMentionSuggestions(state: MessageInputState) {
+        (mentionSuggestionsContent as? MessageComposerChild)?.renderState(state)
+
+        val popupWindow = suggestionsPopup ?: createSuggestionPopupWindow()
+        popupWindow.apply {
+            mentionSuggestionsContent.measure(
+                MeasureSpec.makeMeasureSpec(Resources.getSystem().displayMetrics.widthPixels,
+                    MeasureSpec.EXACTLY),
+                MeasureSpec.makeMeasureSpec(0, MeasureSpec.UNSPECIFIED)
+            )
+            val offset = mentionSuggestionsContent.measuredHeight + this@MessageComposerView.height
+            if (isShowing) {
+                update(this@MessageComposerView, 0, -offset, -1, -1)
```

Comment:
Could we add named arguments here? (maybe we can't if it's calling Java code)

Reply:
it's not possible here :(